node/tessel.js

```diff
@@ -93,24 +93,30 @@ Tessel.Port = function(name, socketPath, board) {
           throw new Error('Received an unexpected response with no commands pending: ' + byte);
         }
 
-        var data_size = this.replyQueue[0].size;
+        var queued = this.replyQueue.shift();
```

Comment:
Whoops–this was a last minute change. I will add tests that ensure it's left intact.